hilti/runtime/include/fiber.h

```diff
@@ -124,10 +115,16 @@ class Fiber {
     std::optional<std::any> _result;
     std::exception_ptr _exception;
 
-    ucontext_t _uctx{};
-    jmp_buf _fiber{};
-    jmp_buf _trampoline{};
-    jmp_buf _parent{};
+    /** The underlying coroutine of this fiber. */
+    ::Fiber _fiber;
+
+    /**
+     * The coroutine this fiber yields to.
+     *
+     * This is typically the coroutine of the fiber which invoked `run` on this
+     * coroutine.
+     */
+    ::Fiber* _parent = nullptr;
```

Comment:
Is the parent for a fiber always the main fiber? If so, do we need store it given that have it globally available now?

Reply:
No, `_parent` is the fiber we yield to and is the fiber that triggered us.